situp-benchmarks/mapdb-benchmarks/README.md

```diff
@@ -0,0 +1,17 @@
+# MapDb Benchmarks
```

Comment:
why separate subproject can it not be part of ```:situp-plugins:mapdb-processor-state``` ?

Reply:
I had asked the team, and most people seemed to be aligned with making this a separate sub project. its not testing logic of the class and its not part of the source code for that project. it also seems like keeping the benchmarks together in a subproject would be good for running benchmark suites together